bindings/nodejs/native/src/classes/account_manager/mod.rs

```diff
@@ -149,7 +148,6 @@ declare_types! {
             let signer_type: AccountSignerType = serde_json::from_str(&signer_type.to_string()).expect("invalid signer type");
             let signer_type = match signer_type {
```

Comment:
Isn't that match useless now?

Reply:
I left it because we'll add ledger signing soon